### ISSUE TITLE
New Version 1.5.8 is not installable

### ISSUE THREAD

**Issue by mburger81** [silkimen/cordova-plugin-advanced-http#5]
On trying to install latest version with this command

cordova plugin add cordova-plugin-advanced-http@1.5.8 --save

we get this error
```
[ERROR] An error occurred while running cordova plugin add cordova-plugin-advanced-http@1.5.8 --save (exit code 1):

        
        Installing "cordova-plugin-advanced-http" for android
        Installing "cordova-plugin-file" for android
        Installing "cordova-plugin-compat" for android
        ANDROID_HOME=/home/abc/software/android/sdk
        JAVA_HOME=/usr/lib/jvm/jdk1.8.0_121
        Subproject Path: CordovaLib
        The Task.leftShift(Closure) method has been deprecated and is scheduled to be removed in Gradle 5.0. Please use 
        Task.doLast(Action) instead.
                at build_hglad901zbfl7zbg5bgl3vk5.run(/home/abc/tmp/ionic-3.8.1/platforms/android/build.gradle:137)
        The JavaCompile.setDependencyCacheDir() method has been deprecated and is scheduled to be removed in Gradle 4.0.
        Incremental java compilation is an incubating feature.
        The TaskInputs.source(Object) method has been deprecated and is scheduled to be removed in Gradle 4.0. Please 
        use TaskInputs.file(Object).skipWhenEmpty() instead.
        :clean
        :CordovaLib:clean
        
        BUILD SUCCESSFUL
        
        Total time: 0.9 secs
        
        The Android Persistent storage location now defaults to "Internal". Please check this plugin's README to see if 
        your application needs any changes in its config.xml.
        
        If this is a new application no changes are required.
        
        If this is an update to an existing application that did not specify an "AndroidPersistentFileLocation" you may 
        need to add:
        
               "<preference name="AndroidPersistentFileLocation" value="Compatibility" />"
        
        to config.xml in order for the application to find previously stored files.
                
        Error during processing of action! Attempting to revert...
        Failed to install 'cordova-plugin-advanced-http': CordovaError: Uh oh!
        "/home/abc/tmp/ionic-3.8.1/plugins/cordova-plugin-advanced-http/src/android/com/synconset/cordovahttp/CordovaHttp.java" 
        not found!
             at copyFile (/home/abc/tmp/ionic-3.8.1/platforms/android/cordova/lib/pluginHandlers.js:219:36)
             at copyNewFile (/home/abc/tmp/ionic-3.8.1/platforms/android/cordova/lib/pluginHandlers.js:250:5)
             at install (/home/abc/tmp/ionic-3.8.1/platforms/android/cordova/lib/pluginHandlers.js:43:17)
             at Object.process 
        (/home/abc/tmp/ionic-3.8.1/platforms/android/cordova/node_modules/cordova-common/src/ActionStack.js:56:25)
             at PluginManager.doOperation 
        (/home/abc/tmp/ionic-3.8.1/platforms/android/cordova/node_modules/cordova-common/src/PluginManager.js:117:20)
             at PluginManager.addPlugin 
        (/home/abc/tmp/ionic-3.8.1/platforms/android/cordova/node_modules/cordova-common/src/PluginManager.js:147:17)
             at /home/abc/tmp/ionic-3.8.1/platforms/android/cordova/Api.js:254:18
             at _fulfilled (/home/abc/tmp/ionic-3.8.1/platforms/android/cordova/node_modules/q/q.js:854:54)
             at self.promiseDispatch.done 
        (/home/abc/tmp/ionic-3.8.1/platforms/android/cordova/node_modules/q/q.js:883:30)
             at Promise.promise.promiseDispatch 
        (/home/abc/tmp/ionic-3.8.1/platforms/android/cordova/node_modules/q/q.js:816:13)
        Error: Uh oh!
        "/home/abc/tmp/ionic-3.8.1/plugins/cordova-plugin-advanced-http/src/android/com/synconset/cordovahttp/CordovaHttp.java" 
        not found!

```

With version before which is 1.3.7 anything is working without error

**Comment by silkimen** [comment]
Which OS are you using on your developer machine? Do you have a case-sensistive filesystem?
Is the file "CordovaHttp.java" really missing or maybe is the path wrong?

e.g. "/src/android/com/synconset/**CordovaHttp**/CordovaHttp.java" instead of "/src/android/com/synconset/**cordovahttp**/CordovaHttp.java"

**Comment by mburger81** [comment]
Yes I have a case sensitive filesystem. I'm using an ubuntu system. 
I can not check for the url, I can do it only on Monday morning. I'll let you know on Monday

**Comment by silkimen** [comment]
Ok, I found the issue. Will fix it!

**Comment by silkimen** [comment]
fixed in **v1.5.9**
